src/common/module.c

```diff
@@ -40,7 +40,8 @@ static int run_command(const char *command)
 
 	if (status < 0) {
 		char error_buf[80];
-		strerror_r(errno, error_buf, sizeof(error_buf));
+		char* dummy = strerror_r(errno, error_buf, sizeof(error_buf));
```

Comment:
Actually I've tried both before posting this pr but none of them worked.

By applying @Liuchang0812  change, I get:
```
/home/xxg/dev/ceph-dev/src/common/module.c: In function ‘run_command’:
/home/xxg/dev/ceph-dev/src/common/module.c:43:8: warning: type defaults to ‘int’ in declaration of ‘dummy’ [enabled by default]
   auto dummy = strerror_r(errno, error_buf, sizeof(error_buf));
        ^
/home/xxg/dev/ceph-dev/src/common/module.c:43:16: warning: initialization makes integer from pointer without a cast [enabled by default]
   auto dummy = strerror_r(errno, error_buf, sizeof(error_buf));
                ^
```
and by applying @tchaikov  change instead I get:

```
/home/xxg/dev/ceph-dev/src/common/module.c: In function ‘run_command’:
/home/xxg/dev/ceph-dev/src/common/module.c:44:3: warning: ignoring return value of ‘strerror_r’, declared with attribute warn_unused_result [-Wunused-result]
   (void)strerror_r(errno, error_buf, sizeof(error_buf));
   ^
```